jack-sys/src/lib.rs

```diff
@@ -512,6 +518,101 @@ external_library!(
     jack_info_callback: Option<unsafe extern "C" fn(*const ::libc::c_char) -> ()>,
 );
 
+external_library!(
+    JackCtl,
+    "jack",
```

Comment:
```suggestion
    JACK_LIB,
```
Doesn't this need the platform-specific name?

Reply:
No, this acts similar to the link macro that was replaced.

```rust
#[link(name = "jack")]
extern "C" {
    ...
}
```

Reply:
Works fine for Linux. May need something special for windows.

TODO: Try it out on Windows. According to the readme, this has worked on windows so maybe:

```rust
#[link(name = "C:/Program Files/JACK2/lib/libjack64")]
```

Reply:
Hmm, I don't see any [documentation](https://doc.rust-lang.org/nomicon/ffi.html#linking) how the `#[link]` name is used to locate the library...

Reply:
https://doc.rust-lang.org/rustc/command-line-arguments.html#-l-link-the-generated-crate-to-a-native-library

> The name used in a link attribute may be overridden using the form -l ATTR_NAME:LINK_NAME where ATTR_NAME is the name in the link attribute, and LINK_NAME is the name of the actual library that will be linked.

Perhaps that's how the platform-specific file name should be specified [in build.rs](https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorustc-link-libkindname)?

Reply:
I have an idea to clean up the platform-specific stuff... working on a PR now.

Reply:
That idea didn't work out. I think the best way to handle these platform-specific library names is to pass the `JACK_LIB` static to the `external_library!` macros as I originally suggested. Changing the Linux one to `libjack.so.0` rather than `libjack.so` fixes the issue of needing the `-devel` package installed.